MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -88,11 +88,15 @@ internal void SetVertexAttributeArray(bool[] attrs)
             }
         }
 
-        private void PlatformSetup()
+       private void PlatformSetup()
         {
 #if WINDOWS || LINUX || ANGLE
             GraphicsMode mode = GraphicsMode.Default;
-            var wnd = (Game.Instance.Window as OpenTKGameWindow).Window.WindowInfo;
+
+            var wnd = OpenTK.Platform.Utilities.CreateWindowsWindowInfo(PresentationParameters.DeviceWindowHandle);
```

Comment:
I don't know how safe it is with instancing, I would have to look at the code, but from memory, I don't think that this will work on linux or mac(there are methods like CreateX11WindowInfo and CreateMacWindowInfo for them).

Doing something like the code bellow would make much more sence:
IWindowInfo wnd;
if(Game.Instance.Window as OpenTKGameWindow != null)
    wnd = (Game.Instance.Window as OpenTKGameWindow).Window.WindowInfo;
else
    wnd = OpenTK.Platform.Utilities.CreateWindowsWindowInfo(PresentationParameters.DeviceWindowHandle);